app/helpers/link_to_function_helper.rb

```diff
@@ -0,0 +1,11 @@
+module LinkToFunctionHelper
+  def link_to_function(name, *args, &block)
+     html_options = args.extract_options!.symbolize_keys
+
+     function = block_given? ? update_page(&block) : args[0] || ''
+     onclick = "#{"#{html_options[:onclick]}; " if html_options[:onclick]}#{function}; return false;"
+     href = html_options[:href] || '#'
+
+     content_tag(:a, name, html_options.merge(:href => href, :onclick => onclick))
```

Comment:
Om vi kör med en version av ruby där hash-rockets inte är ett krav tycker jag vi bör hålla oss ifrån dem om de inte behövs.

```suggestion
     content_tag(:a, name, html_options.merge(href: href, onclick: onclick))
```

Reply:
@alex-ross Sant (på alla punkter), ändrade och kör merge på det strax 👍